android/app/src/main/java/app/organicmaps/util/Config.java

```diff
@@ -59,6 +59,12 @@ public final class Config
 
   private Config() {}
 
+  @SuppressWarnings("ConstantConditions") // BuildConfig
```

Comment:
I retained this but I don't know whether this is still necessary as I don't have a full IDE at present.